.travis.yml

```diff
@@ -28,8 +32,9 @@ before_script:
   - mkdir -p ~/.config/satnogs-gut
 
 script:
-  - cd station && pytest -v && cd ..
-  - cd server && PYTHONTEST=. pytest -v
+  - cd station && pytest -v ; cd ..
+  - cd server && PYTHONTEST=. pytest -v ; cd ..
+  - shellcheck ${SHELL_SCRIPTS} ${SHELLCHECK_OPTS}
```

Comment:
Could we use here the wildcard "*.sh"? Is it supported?

Reply:
Sure, we can. This will require fixing several scripts, though. Not a bit problem, though.

Reply:
Ok, then I accept your MR and add an issue for improve shell scripts.